wesnoth-wof.po

```diff
@@ -482,7 +549,7 @@ msgstr ""
 #. [message]: speaker=Reshan
 #: data/campaigns/Winds_of_Fate/scenarios/01_The_Hunt.cfg:557
 msgid "Victory is yours, <b>Aspirant</b> Gorlack."
-msgstr ""
+msgstr "Vítězství je tvé, <b>Aspirante</b> Gorlaku."
```

Comment:
```suggestion
msgstr "Vítězství je tvé, <b>aspirante</b> Gorlaku."
```

Reply:
Je pravda, že pro zdůraznění asi stačí to tučné písmo. Nebo to dát celé velkými písmeny?

Reply:
Také si myslím, že tučné je dostatečné.

Všechna velká písmena se obvykle používají při vyjádření křiku.